docs/getting-started/how-to/add-github-codespaces-to-your-project/index.md

```diff
@@ -0,0 +1,126 @@
+This page describes how to add GitHub Codespaces to your project.
+
+## Does your project already have GitHub Codespaces?
+
+Newer projects already have GitHub Codespaces.
+However, older projects don't.
+To find out whether your project is newer or older:
+
+* Click on your project in the [opensafely](https://github.com/opensafely) organisation on GitHub.
+* Can you see a `.devcontainer` folder?
+  If not, then your project is **older**.
+* Click on the `.devcontainer` folder to open it.
+* Click on the `devcontainer.json` file to open it.
+* Can you see the following line towards the top of the file?
+  ```json
+  "image": "ghcr.io/opensafely-core/research-template:v0",
+  ```
+  If not, then your project is **older**.
```

Comment:
I understand why you've written it in this order, but I think the rendered output looks better as this: 

* Can you see the following line towards the top of the file?  If not, then your project is **older**.
  ```json
  "image": "ghcr.io/opensafely-core/research-template:v0",
  ```

wrt the following bullet point, as opposed to have indented but un-bulleted text in-between

Reply:
Reordered! Thank you for the suggestion.